tap/tcp_stream.go

```diff
@@ -89,6 +90,10 @@ func (t *tcpStream) GetReqResMatchers() []api.RequestResponseMatcher {
 }
 
 func (t *tcpStream) GetIsTapTarget() bool {
+	if os.Getenv("MIZU_TAPPER_NO_TAP_TARGET") == "true" {
+		return false
+	}
```

Comment:
This will halt TCP reassembly in a bad place:

https://github.com/up9inc/mizu/blob/4ec98ff76bb7130f69dae9eb874236c342ad7de3/tap/tcp_reassembly_stream.go#L135-L147

Such that it will cause a lot of hanging dissector Goroutines.

Reply:
Shouldn't be a problem because of https://github.com/up9inc/mizu/blob/11e1286286e547813febffba90bb335387f02790/tap/tcp_stream_factory.go#L62. The routines won't be created to begin with.